### PR TITLE
Bump bimap upper bound to <0.6

### PR DESCRIPTION
**Pull request by carbolymer**
# Description
Prerequisite of: https://github.com/input-output-hk/cardano-api/pull/80

### CODE REVIEW

File: ouroboros-consensus-cardano/ouroboros-consensus-cardano.cabal [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cabal-version:   3.0
 name:            ouroboros-consensus-cardano
-version:         0.6.1.0
```
Comment (dnadales) [inline ouroboros-consensus-cardano/ouroboros-consensus-cardano.cabal:3]: This is not necessary.

**Review by dnadales — CHANGES_REQUESTED** [review]
Thanks for the PR. We should not update the package version though.

File: ouroboros-consensus-cardano/ouroboros-consensus-cardano.cabal (hunk 1 above)
Comment (carbolymer) [reply ouroboros-consensus-cardano/ouroboros-consensus-cardano.cabal]: removed

File: ouroboros-consensus/ouroboros-consensus.cabal [hunk 2]
```diff
@@ -1,6 +1,6 @@
 cabal-version:   3.0
 name:            ouroboros-consensus
-version:         0.8.0.0
```
[the remark this answers is not in the document]
Comment (carbolymer) [reply ouroboros-consensus/ouroboros-consensus.cabal]: removed

**Review by dnadales — APPROVED** [review]

**Pull request merged**